data/ut/legislature/Lincoln-Fillmore-04f24031-8ac2-42c4-8b6f-dd8a414f18ad.yml

```diff
@@ -0,0 +1,38 @@
+id: ocd-person/04f24031-8ac2-42c4-8b6f-dd8a414f18ad
+name: Lincoln Fillmore
+given_name: Lincoln
+family_name: Fillmore
+email: lfillmore@le.utah.gov
+image: https://le.utah.gov/images/legislator/FILLML.jpg
+party:
+- name: Republican
+roles:
+- start_date: '2019-01-01'
```

Comment:
@sroomf this person should have a previous file we should add their new role to:
-  type: upper
    jurisdiction: ocd-jurisdiction/country:us/state:ut/government
    district: '17'
They were previously in Senate district 10 from Jan 2017-Jan 2023